extensions/ql-vscode/src/databases/db-item-selection.ts

```diff
@@ -80,15 +80,15 @@ export function mapDbItemToSelectedDbItem(
     case DbItemKind.LocalDatabase:
       return {
         kind: SelectedDbItemKind.LocalDatabase,
-        listName: dbItem?.parentListName,
         databaseName: dbItem.databaseName,
+        listName: dbItem?.parentListName,
```

Comment:
Is there a reason we've moved these around?

Reply:
Yes, that's the reason why `JSON.stringify(mappedItem) === JSON.stringify(selectedItem)` is so ugly: it depends on order. There doesn't seem to be an easy way to compare Javascript objects, (or sort them), so that's what we got..

Reply:
this is not needed anymore but I'll leave it so that it's the same order everywhere